lib/project/project-constants.ts

```diff
@@ -20,4 +20,9 @@ export class ProjectConstants implements Project.IProjectConstants {
 		MobileWebsite: "MobileWebsite"
 	};
 }
+
+export class ImageConstants {
```

Comment:
Do you need this to be separate class? You can add ImageDefinitionsFileName inside ProjectConstants class, this way you will be able to use `projectConstants` from constructor injection and this https://github.com/Icenium/icenium-cli/pull/1147/files#diff-dabbf21200eb074fe5d07cc5a33ad85cR7 will be unneeded.